### DISCUSSION TITLE
apoyo

### DISCUSSION THREAD

**Discussion by devbastian08** [coreybutler/nvm-windows#990] [Support]
hola, tengo un pc con win7, el problema que tengo es que al escribir el comando nvm on , me salta una advertencia de seguridad que dice no se puede abir estos archivos 
![image](https://github.com/coreybutler/nvm-windows/assets/119069676/2cbf36fc-64a3-4f1a-a9e3-387766dd45eb)

**Comment by coreybutler** [comment]
## Translated from Spanish:

Hello, I have a PC with Win7, the problem I have is that when I type the command NVM ON, I get a security warning that says you can not open these files

# Answer
You probably need to run as an administrator. When `nvm on` is used, it attempts to run `nvm use` to configure the initial node version. This requires setting a symlink. Windows 7 doesn't support non-admin symlinks.

Please note, NVM4W probably works in Windows 7, but isn't tested or supported in anything less than the last two major releases (i.e. Windows 10 and 11).

## Translated to Spanish:
Probablemente necesite ejecutar como administrador. Cuando se utiliza 'nvm on', intenta ejecutar 'nvm use' para configurar la versión inicial del nodo. Esto requiere establecer un enlace simbólico. Windows 7 no admite vínculos simbólicos que no sean de administrador.

Tenga en cuenta que NVM4W probablemente funciona en Windows 7, pero no está probado ni es compatible con nada menos que las dos últimas versiones principales (es decir, Windows 10 y 11).

**Reply by devbastian08** [reply]
que version de node.j es compatible con 32 bits?

**Reply by coreybutler** [reply]
English Translation: Which versions of Node.js are compatible with 32-bits?

**Answer:**
Pretty much all of them. Run `nvm install x.x.x 32` to install a 32-bit version.

_Spanish Translation:_ Casi todos. Ejecute 'nvm install x.x.x 32' para instalar una versión de 32 bits.
